prometheus-to-sd/config/source_config_test.go

```diff
@@ -31,29 +32,32 @@ func TestNewSourceConfig(t *testing.T) {
 		host        string
 		port        string
 		whitelisted string
+		config      string
 		output      SourceConfig
 	}{
-		{"testComponent", "localhost", "1234", "a,b,c,d",
+		{"testComponent", "localhost", "1234", "a,b,c,d", "",
```

Comment:
I'm not sure what to do about the tests.

Reply:
That should be OK.